mcorpc/authz_rego_test.go

```diff
@@ -0,0 +1,95 @@
+package mcorpc
+
+import (
+	"github.com/choria-io/go-choria/choria"
+	"github.com/choria-io/go-choria/server/agents"
+	"github.com/choria-io/go-config"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/sirupsen/logrus"
+)
+
+var _ = Describe("RegoPolicy", func() {
+	var (
+		authz  *regoPolicy
+		pol    *regoPolicyPolicy
+		logger *logrus.Entry
+		fw     *choria.Framework
+		err    error
+	)
+
+	BeforeEach(func() {
+		// logbuffer = &bytes.Buffer{}^
+		logger = logrus.NewEntry(logrus.New())
+		logger.Logger.Out = GinkgoWriter
```

Comment:
can also do `ioutil.Discard` once we dont want to see things in tests

Reply:
@ripienaar the log data only shows up when you pass -v to ginkgo?

Reply:
TIL :)

That’s pretty nice. Might need to mass change tests now doh

Reply:
some of your tests you examine the log output though, but in general .. yeah, should switch it over to this.